styles/modern.css

```
@@ -107,7 +107,7 @@ h6 {
 }
 
 .title-font {
-    font-family: 'outline-offset', -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
+    font-family: 'outline-offset', 'Noto Sans JP', 'Segoe UI', 'Microsoft YaHei', 'Meiryo', 'Apple SD Gothic Neo', 'PingFang SC', 'Hiragino Sans', -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "outline-offset" (font-family-name-quotes)

(font-family-name-quotes)